src/pages/assessment/AssessmentsTable.tsx

```diff
@@ -89,13 +89,21 @@ export const AssessmentsTable: React.FC<Props> = ({
       // Parse snapshot data using utility function
       const snapshotData = parseLatestSnapshot(snapshots);
 
+      // Format owner name from first and last name
+      const ownerFirstName = assessmentModel?.ownerFirstName;
+      const ownerLastName = assessmentModel?.ownerLastName;
```

Comment:
Both first name and last name start with a majuscule?

Reply:
I don't understand the question, can you please explain it?

Reply:
you need to make sure that each word starts with a capital letter. like "John Doe".

Reply:
I added format code to make sure the names have proper capitalization
Please review this again